### ISSUE TITLE
Most intrinsic functions appear to be unsupported

### ISSUE THREAD

**Issue by Makeshift** [aws/amazon-states-language-service#85]
Most intrinsic functions that are supported by ASL appear as an error when using the language service with the [VSCode AWS Toolkit](https://github.com/aws/aws-toolkit-vscode) extension.

I believe these intrinsic functions are fairly new, so I'm assuming nobody has added validators for them yet.

Here's a test Step Function showing which ones work with the ASL service and which ones don't

```yaml
Comment: Check if the ASL Service can validate all of the intrinsic functions.
StartAt: Set some test vars
States:
  Set some test vars:
    Type: Pass
    Parameters:
      foo: bar
      baz:
        quz: lol
      bar:
        bub: boo
      o: o
      boo:
        - 1
        - 2
        - 99
      base: Zm9v
      hash: MD5
      str: '{"foo": "bar"}'
    Next: Test
  Test:
    Type: Pass
    Parameters:

      Supported:
        Array.$: 'States.Array($.foo, $.baz.quz, $.boo)'
        Format.$: States.Format('formatting {}', $.foo)
        JsonToString.$: States.JsonToString($.baz)
        StringToJson.$: States.StringToJson($.str)

      Unsupported:
        ArrayContains.$: 'States.ArrayContains($.boo, $.boo[0])'
        ArrayGetItem.$: 'States.ArrayGetItem($.boo, 0)'
        ArrayLength.$: States.ArrayLength($.boo)
        ArrayPartition.$: 'States.ArrayPartition($.boo, 1)'
        ArrayRange.$: 'States.ArrayRange(1, 9, 2)'
        ArrayUnique.$: States.ArrayUnique($.boo)
        Base64Decode.$: States.Base64Decode($.base)
        Base64Encode.$: States.Base64Encode($.foo)
        Hash.$: 'States.Hash($.foo, $.hash)'
        JsonMerge.$: 'States.JsonMerge($.baz, $.bar, false)'
        MathAdd.$: 'States.MathAdd($.boo[0], $.boo[1])'
        MathRandom.$: 'States.MathRandom($.boo[0], $.boo[2])'
        StringSplit.$: 'States.StringSplit($.bar.bub, $.o)'
        UUID.$: States.UUID()

    End: true
```

Same, but in JSON:

```json
{
  "Comment": "Check if the ASL Service can validate all of the intrinsic functions.",
  "StartAt": "Set some test vars",
  "States": {
    "Set some test vars": {
      "Type": "Pass",
      "Parameters": {
        "foo": "bar",
        "baz": {
          "quz": "lol"
        },
        "bar": {
          "bub": "boo"
        },
        "o": "o",
        "boo": [
          1,
          2,
          99
        ],
        "base": "Zm9v",
        "hash": "MD5",
        "str": "{\"foo\": \"bar\"}"
      },
      "Next": "Test"
    },
    "Test": {
      "Type": "Pass",
      "Parameters": {
        "Supported": {
          "Array.$": "States.Array($.foo, $.baz.quz, $.boo)",
          "Format.$": "States.Format('formatting {}', $.foo)",
          "JsonToString.$": "States.JsonToString($.baz)",
          "StringToJson.$": "States.StringToJson($.str)"
        },
        "Unsupported": {
          "ArrayContains.$": "States.ArrayContains($.boo, $.boo[0])",
          "ArrayGetItem.$": "States.ArrayGetItem($.boo, 0)",
          "ArrayLength.$": "States.ArrayLength($.boo)",
          "ArrayPartition.$": "States.ArrayPartition($.boo, 1)",
          "ArrayRange.$": "States.ArrayRange(1, 9, 2)",
          "ArrayUnique.$": "States.ArrayUnique($.boo)",
          "Base64Decode.$": "States.Base64Decode($.base)",
          "Base64Encode.$": "States.Base64Encode($.foo)",
          "Hash.$": "States.Hash($.foo, $.hash)",
          "JsonMerge.$": "States.JsonMerge($.baz, $.bar, false)",
          "MathAdd.$": "States.MathAdd($.boo[0], $.boo[1])",
          "MathRandom.$": "States.MathRandom($.boo[0], $.boo[2])",
          "StringSplit.$": "States.StringSplit($.bar.bub, $.o)",
          "UUID.$": "States.UUID()"
        }
      },
      "End": true
    }
  }
}
```

Errors shown in VSCode (From the ASL service):
![image](https://user-images.githubusercontent.com/345013/195432243-474566b4-5119-4db0-aea9-ecded0dec859.png)

(Same errors show for the JSON version)

Works as expected in an actual Step Function:
![image](https://user-images.githubusercontent.com/345013/195432446-2821b87e-1ff0-48aa-b51b-fd9ef174d6e6.png)

In summary, these functions need to be supported:
- ArrayContains
- ArrayGetItem
- ArrayLength
- ArrayPartition
- ArrayRange
- ArrayUnique
- Base64Decode
- Base64Encode
- Hash
- JsonMerge
- MathAdd
- MathRandom
- StringSplit
- UUID

Unfortunately, using any of the above intrinsic functions means that the toolkit can't render a graph. Is there a way to tell the ASL server to ignore the next line, or any other setting that can help here until support is added?

**Comment by jgroom33** [comment]
This behavior also exists when using the [simulator](https://us-east-1.console.aws.amazon.com/states/home?region=us-east-1#/simulator)
